cypress/data/data.ts

```diff
@@ -0,0 +1,17 @@
+export const env = {
+  heroku: "https://montelearn-app.herokuapp.com/",
+};
+
+export const users = {
+  monteTeacher: {
+    username: "teacher@test.com",
+    password: "Eifi6eeChu",
+  },
+  monteStudent: {
+    username: "student@test.com",
+    password: "baeng3MeeB",
+  },
+};
```

Comment:
We can't store passwords in the code, I think that it should an environment variable

Reply:
aint that just a mock data tho?